python/python/glide/async_commands/sorted_set.py

```diff
@@ -169,6 +170,153 @@ def __init__(
         self.limit = limit
 
 
+class GeospatialData:
+    def __init__(self, longitude: float, latitude: float):
+        """
+        Represents a geographic position defined by longitude and latitude.
+
+        The exact limits, as specified by EPSG:900913 / EPSG:3785 / OSGEO:41001 are the following:
+            - Valid longitudes are from -180 to 180 degrees.
+            - Valid latitudes are from -85.05112878 to 85.05112878 degrees.
+
+        Args:
+            longitude (float): The longitude coordinate.
+            latitude (float): The latitude coordinate.
+        """
+        self.longitude = longitude
+        self.latitude = latitude
+
+
+class GeoUnit(Enum):
+    """
+    Enumeration representing distance units options for the `GEODIST` command.
+    """
+
+    METERS = "m"
+    """
+    Represents distance in meters.
+    """
+    KILOMETERS = "km"
+    """
+    Represents distance in kilometers.
+    """
+    MILES = "mi"
+    """
+    Represents distance in miles.
+    """
+    FEET = "ft"
+    """
+    Represents distance in feet.
+    """
+
+
+class OrderBy(Enum):
+    """
+    Enumeration representing sorting order options for various commands.
+
+    This enum is used for the following commands:
+    - `SORT`: General sorting in ascending or descending order.
+    - `GEOSEARCH`: Sorting items based on their proximity to a center point.
```

Comment:
Pretty big difference in the meaning. In that case it is OK to create 2 enums, with different names: `SortOrderBy` and `GeoOrderBy` or similar.

Reply:
I think it is okay to use the same enum' we might need a bette doc 🤔

Reply:
I would remove this section.  This documentation assumes knowledge of how calling commands will use this, but this documentation should exist in the commands.  

GEOSEARCH command should have a note that says ASC is nearest distance to center point, etc...

Reply:
so I will just leave what commands this is used for